### ISSUE TITLE
Unexpected behaviour when highlighting text

### ISSUE THREAD

**Issue by tnajdek** [zotero/zotero#4574]
Seems to be specific to this one PDF attached. 

https://github.com/user-attachments/assets/4c91f450-b22c-4615-b6e6-cc471e05d7e9

[Fernandez-Gimenez et al. - 2008 - Adaptive M.pdf](https://github.com/user-attachments/files/16623852/Fernandez-Gimenez.et.al.-.2008.-.Adaptive.M.pdf)

**Comment by Sunbuyu** [comment]
Yes, I encountered this problem, too.  This happens only when highlighting the lines with line number. The pdf file is generated by latex. For those lines without line number, it works just fine.

**Comment by mrtcode** [comment]
@tnajdek It seems the PDF file differs from the one in the video. I'm also unable to reproduce the issue using similar files.

**Comment by tnajdek** [comment]
Sorry, must have uploaded wrong PDF. I've retested (with the PDF from the video) and can no longer reproduce. I think we can close this one!
